.changeset/olive-cups-kiss.md

```diff
@@ -37,4 +37,4 @@ The following option flags are now available in their respective scope
 
 #### Node version change
 
-Moved to node `v20.18.1`
+Moved to node `v20.15`
```

Comment:
This is a significant enough change that it should be in its own changeset and have a CTA for consumers to update their node version. See [here](https://github.com/seek-oss/sku/pull/986/files#diff-495e18c37d72bd31e3e6052dc0d76b80d6102de62f8e25e5eea3d984dd2ea29d) for prior art.